docs/airgap.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 20
+keywords:
+- Harvester
+- offline
+- Air-gap
+- HTTP proxy
+---
+# Air Gapped Environment
+
+This section is about using Harvester in an air gapped environment. An air gapped environment could be where Harvester will be installed offline, behind a firewall, or behind a proxy.
+
+Harvester works in air gapped environment. Everything required to set up Harvester is packaged in the Harvester ISO image.
+
+## Working Behind an HTTP Proxy
+
+In some environments, servers or VMs running on premise can connect to external services through an HTTP(S) proxy.
+
+### Configure an HTTP Proxy During Installation
+
+You can set the HTTP proxy that is used during installation. For example, you can get remote SSH keys or Harvester configuration file via the HTTP proxy.
```

Comment:
```suggestion
You can configure the HTTP(S) proxy during the [ISO installation](./install/iso-install.md) as shown in picture below:
```

Reply:
I combined the two phrases as they were very similar.

I also removed the example as I'm not sure if it makes sense to have it here.